AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -331,6 +331,7 @@ forbes.com#@#.AD-label300x250
 forbes.com#@#.AD-Rotate
 !
 @@||static.criteo.net/images/pixel.gif$domain=gmx.net
+osxinfo.net##body > div[class][style^="background-color: rgba(0, 0, 0, 0.9"][style*="background-position: initial initial;"]
```

Comment:
not working.
Change rule or replace by scripltet and generichide for specific platforms.
For example `osxinfo.net#%#//scriptlet("prevent-setTimeout", "$", "1000")`
Also you can try to block URL using regexp
osxinfo.net/js/o6kYnO1/pE8lJyna6Rgfayx.js

Reply:
Blocking of script doesn't help.
Looks like the rule `osxinfo.net##body > div[class][style*="background"][style*="rgba(0, 0, 0, 0.9"]` works good on all platforms